app-drawer/app-drawer.html

```diff
@@ -311,9 +337,22 @@
         this._setPosition(this.align);
       },
 
+      _focusHandler: function(event) {
+        if (!this.contains(Polymer.dom(event).rootTarget)) {
```

Comment:
`this.contains` will have different results between ShadyDOM and ShadowDOM.  If you want to  check if an element is in the light DOM tree use `isLightDescendant`